dynamic-property-zk/src/test/kotlin/ru/fix/dynamic/property/zk/ZkDynamicPropertySourceTest.kt

```diff
@@ -180,14 +180,45 @@ class ZkDynamicPropertySourceTest {
     @Test
     fun shouldGetDefaultValueFromHolder() {
         val holder = SourcedProperty(
-                source!!,
+                source,
                 "unknown.property",
                 String::class.java,
                 OptionalDefaultValue.of("default Value")
         )
         assertEquals("default Value", holder.get())
     }
 
+    @Test
+    fun `should return actual value instead of default when a large number of properties loaded`() {
+        val generatedProperties = generateProperties(200)
+
+        generatedProperties.forEach {
+            setServerProperty("${PROPERTIES_LOCATION}/${it.key}", it.value)
+        }
+
+        val source = ZkDynamicPropertySource(
+                zkTestingServer.createClient(),
+                PROPERTIES_LOCATION,
+                JacksonDynamicPropertyMarshaller(),
+                Duration.of(1, ChronoUnit.MINUTES)
+        )
+
+        generatedProperties.forEach {
+            source.subscribeAndCallListener(
+                    it.key,
+                    String::class.java,
+                    OptionalDefaultValue.of("default")
+            ) { value ->
+                assertNotEquals("default", value)
+            }
+        }
+
+        source.close()
+    }
+
+    private fun generateProperties(count: Int): Map<String, String> {
```

Comment:
Method signature implies that `generateProperties(10)`  generates 10 properties. Actually you are generating 11 =(